Bottom: 8473a766eb884ac93de75b3ee9b51d0c881449c1
Top:    b69ad1b4ba77d8c88e2f84e735ea909d7ee91fbc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-11 22:46:43 +0100

Refresh of change-scroll-wheel-bindings-t

---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index 7dee9b9..b06482f 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 
   scroll-left = {
     Scroll(left)
